src/main/java/me/ajaja/module/footprint/domain/Ajaja.java

```diff
@@ -0,0 +1,26 @@
+package me.ajaja.module.footprint.domain;
+
+import java.beans.ConstructorProperties;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Getter;
+import me.ajaja.global.common.SelfValidating;
+
+@Getter
+public class Ajaja extends SelfValidating<Ajaja> {
+	@NotNull
+	private final Long userId;
+
+	@NotBlank
+	@Size(max = 20)
+	private final String nickName;
```

Comment:
닉네임은 어떤 이유때문에 들어가나용?.?

Reply:
실제로 화면 조회에 사용한다면 ID 값보다는 사용자 이름이 더 필요하다고 생각했습니다!
근데 지금 Ajaja보니깐 애초에 사용자 이름이 없네유!!
당시에 아좌좌선택한 사용자 정보도 나중에 필요하지 않을까 생각했는데 불필요한 고민이였던 것 같아요!!
삭제도록 하겠습니다.

Reply:
nickname은 합성어가 아닙니다 ㅋㅋ
근데 밑에 리뷰하다 보니까 다른 곳에는 잘 써놓으셧군요?

```suggestion
	@NotBlank
	@Size(max = 20)
	private final String nickname;
```

Reply:
완